src/CodeViewer/include/CodeViewer/Viewer.h

```diff
@@ -71,6 +74,9 @@ class Viewer : public QPlainTextEdit {
 
   FontSizeInEm heatmap_bar_width_ = FontSizeInEm{0.0f};
   HeatmapSource heatmap_source_;
+
+  bool highlight_current_line_ = false;
+  void HighlightCurrentLine();
```

Comment:
Please move this private method near the others

Reply:
Done.